docs/pages/docs/faq.mdx

```diff
@@ -52,6 +52,174 @@ defined and incompatible interfaces - which are the biggest problems in common
+time to prove things. Proving through dependent types shifts some of this burden
+to the type system, but still demand an advanced understanding. Model checking
+(as in Quint) will usually provide weaker guarantees in comparison, but the
+proofs are fully automated.
+
+Another difference is that programs in Agda and Idris need to be total - that
+is, they need to terminate, and the writer may need to convince the type system
+that in fact it does terminate every time.
+
+</FAQBox>
+
+<FAQBox title="How does Quint compare to Abstract State Machines?"> 
+
+Abstract State Machines (ASMs) are a precise way to write pseudocode to define
+the requirements of a system, and they are mostly used to reason about and
+analyze designs, while Quint is more focused on executing and verifying
+specifications.
+
+This [report](https://arxiv.org/abs/2301.10875) explores the differences and
+similarities between Quint, TLA+ and ASM in more detail. 
+
+</FAQBox>
+
+<FAQBox title="How does Quint compare to tools for specific programming languages?">
+
+There is a big advantage and a big disadvantage for using some
+formal verification tool designed for your programming language of choice instead of using Quint.
+- Advantage: you don't have to write a Quint specification, you can use your own code
+- Disadvantage: the verification capabilities will be extremely limited, because
+  reasoning about generic programming language constructs and concrete program
+  states is incredibly complex.
+
+More generally, the state space of programs are often really large and
+unfeasible for any tool to handle. The solution is to abstract things and work
+on a higher level. Doing so in the same programming language is possible, but
+can be tricky. Using a specific language to do so can help to keep a more
+abstract mindset.
+
+Anything you write in Quint is guaranteed to be compatible with the Temporal
+Logic of Actions (TLA - the logic, not TLA+ the language), and therefore
+verifiable with a model checker. If we were to try and translate arbitrary
+programs to TLA, this would only support a fragment of the language that obeys
+certain rules of the logic. This is also true for other verification techniques,
+and that's why formal verification of generic programs is so limited.
+
+</FAQBox>
+
+<FAQBox title="How does Quint compare to static analyzers?">
+
+Quint is used to reason about behaviors of programs and how things change over
+time. If you can use a static analyzer to search for a class of errors (i.e.
+null checks), you should definitely use that. Use Quint for what you cannot do
+with a static analyzer (which is a lot!).
+
+This is also one of the reasons why Quint has a type system. Although we could
+check typing using invariants, as people often do in TLA+, we can do it
+statically with a regular type checker - and that is much better.
+
+Static analyzers won't be able to tell you if your consensus algorithm does
+consensus as you expect, but Quint will.
```

Comment:
I think this one should explain a bit more what static analyzers can be used for and what they cannot - I think it needs a one sentence explanation at the start that lays out what static analyzers even are, on a very high level